### PR TITLE
main/freeswitch: upgrade freeswitch to 1.6.7, also add perl and perlesl subpackagages

### PR DESCRIPTION
**Pull request by mepholic**
Upgrade freeswitch to 1.6.7.1 and add subpackages for freeswitch-perl and freeswitch-perlesl

### CODE REVIEW

File: main/freeswitch/APKBUILD [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Contributor: Michael Mason <ms13sp@gmail.com>
 # Contributor: Cameron Banta <cbanta@gmail.com>
 pkgname=freeswitch
-pkgver=1.6.6
+pkgver=1.6.7
 pkgrel=2
```
Comment (jirutka) [inline main/freeswitch/APKBUILD]: When bumping `pkgver`, reset `pkgrel` to zero.

File: main/freeswitch/APKBUILD [hunk 2]
```diff
@@ -2,8 +2,8 @@
 # Contributor: Michael Mason <ms13sp@gmail.com>
 # Contributor: Cameron Banta <cbanta@gmail.com>
 pkgname=freeswitch
-pkgver=1.6.6
-pkgrel=2
+pkgver=1.6.7
+pkgrel=1
```
Comment (jirutka) [inline main/freeswitch/APKBUILD]: When bumping pkgver, reset pkgrel to zero.

File: main/freeswitch/APKBUILD [hunk 3]
```diff
@@ -115,6 +117,11 @@ build() {
 	make libfreeswitch.la || return 1
 	# finally we build the rest
 	make -j1 all || return 1
+	
+	# build perlesl module
+	cd "$_builddir"/libs/esl
+	make -j1 perlmod || return 1
```
Comment (jirutka) [inline main/freeswitch/APKBUILD]: `|| return 1` is unnecessary behind the last statement of the function.

Comment (ncopa) [inline main/freeswitch/APKBUILD]: but it does not hurt either.

Comment (jirutka) [inline main/freeswitch/APKBUILD]: Yes, but it’s redundant, so it only adds a noise.

**Comment by mepholic** [comment]
Okay, I squashed the commits down to one, and set the pkgrel to 0. I might add the explicit " || return 1" back, as it won't hurt anything.

File: main/freeswitch/APKBUILD (hunk 3 above)
Comment (Barthalion) [inline main/freeswitch/APKBUILD]: This a nitpick and actually doesn't add any noise, except for you.

**Comment by Barthalion** [comment]
Merged in bf8fe206f1bfbfa9466f12466d0a7647340542a7.

**Pull request closed without merging**